tests/dataframe_tests/util.py

```diff
@@ -0,0 +1,95 @@
+import yaml
```

Comment:
could you update [this file](https://github.com/uc-cdis/tube/blob/a0c0079/tests/README.md) to explain how the "dataframe tests" work + anything that you think is worth documenting - such as what we discussed on slack about how to see the DF contents, how to debug a failed test, and how to write a new test